coin/coins.yml

```diff
@@ -329,3 +329,10 @@
   decimals: 6
   blockTime: 20000
   sampleAddress: '4EZFQABCVQTHQCK3HQBIYGC4NV2VM42FZHEFTVH77ROG4ZGREC6Y7V5T2U'
+
+- id: 165
+  symbol: NANO
+  handle: nano
+  name: Nano
+  decimals: 30
```

Comment:
can you add `blockTime` property too?

Reply:
blockTime might not work for nano, since each account is a blockchain, let's address this later?

Reply:
https://github.com/trustwallet/blockatlas/issues/508